2021-09-22 16:19:01,434 INFO    MainThread:17724 [wandb_setup.py:_flush():69] setting env: {}
2021-09-22 16:19:01,434 INFO    MainThread:17724 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-22 16:19:01,435 INFO    MainThread:17724 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-22 16:19:01,436 INFO    MainThread:17724 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210922_161901-ib3r9r4d\logs\debug.log
2021-09-22 16:19:01,436 INFO    MainThread:17724 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210922_161901-ib3r9r4d\logs\debug-internal.log
2021-09-22 16:19:01,438 INFO    MainThread:17724 [wandb_init.py:init():375] calling init triggers
2021-09-22 16:19:01,481 INFO    MainThread:17724 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-22 16:19:01,482 INFO    MainThread:17724 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 2swy9w42
2021-09-22 16:19:01,482 INFO    MainThread:17724 [wandb_run.py:finish():1271] finishing run khushal/last-weather-recognization/2swy9w42
2021-09-22 16:19:01,489 INFO    MainThread:17724 [jupyter.py:save_history():353] not saving jupyter history
2021-09-22 16:19:01,489 INFO    MainThread:17724 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-22 16:19:01,490 INFO    MainThread:17724 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-09-22 16:19:01,490 INFO    MainThread:17724 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-22 16:19:01,491 INFO    MainThread:17724 [wandb_run.py:_restore():1543] restore
2021-09-22 16:19:02,012 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-09-22 16:19:02,691 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-09-22 16:19:04,709 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 476
}

2021-09-22 16:19:05,567 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:05,671 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:05,778 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:05,883 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:05,986 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,090 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,192 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,295 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,400 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,510 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,616 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,720 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,851 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 476
  total_bytes: 20817
}

2021-09-22 16:19:06,983 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20790
  total_bytes: 20817
}

2021-09-22 16:19:07,086 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:07,190 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:07,293 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:07,397 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:07,499 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:07,602 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:08,796 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:08,900 INFO    MainThread:17724 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 20817
  total_bytes: 20817
}

2021-09-22 16:19:11,096 INFO    MainThread:17724 [wandb_run.py:_show_files():1915] logging synced files
2021-09-22 16:19:11,106 INFO    MainThread:17724 [wandb_init.py:init():424] starting backend
2021-09-22 16:19:11,107 INFO    MainThread:17724 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-22 16:19:11,118 INFO    MainThread:17724 [backend.py:ensure_launched():135] starting backend process...
2021-09-22 16:19:11,332 INFO    MainThread:17724 [backend.py:ensure_launched():139] started backend process with pid: 19732
2021-09-22 16:19:11,333 INFO    MainThread:17724 [wandb_init.py:init():429] backend started and connected
2021-09-22 16:19:11,335 INFO    MainThread:17724 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-09-22 16:19:11,335 INFO    MainThread:17724 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-22 16:19:11,335 INFO    MainThread:17724 [wandb_init.py:init():477] updated telemetry
2021-09-22 16:19:11,337 INFO    MainThread:17724 [wandb_init.py:init():500] communicating current version
2021-09-22 16:19:16,322 INFO    MainThread:17724 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-22 16:19:16,324 INFO    MainThread:17724 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-22 16:19:17,018 INFO    MainThread:17724 [wandb_init.py:init():540] starting run threads in backend
2021-09-22 16:19:22,039 INFO    MainThread:17724 [wandb_run.py:_console_start():1601] atexit reg
2021-09-22 16:19:22,042 INFO    MainThread:17724 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-22 16:19:22,042 INFO    MainThread:17724 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-22 16:19:22,046 INFO    MainThread:17724 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-22 16:19:22,047 INFO    MainThread:17724 [wandb_init.py:init():565] run started, returning control to user process
2021-09-22 16:19:22,047 INFO    MainThread:17724 [wandb_config.py:__setitem__():141] config set epochs = 60 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002267D4E8C70>>
2021-09-22 16:19:22,048 INFO    MainThread:17724 [wandb_run.py:_config_callback():843] config_cb epochs 60 None
2021-09-22 16:19:22,048 INFO    MainThread:17724 [wandb_config.py:__setitem__():141] config set size = 128 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002267D4E8C70>>
2021-09-22 16:19:22,049 INFO    MainThread:17724 [wandb_run.py:_config_callback():843] config_cb size 128 None
2021-09-22 16:19:22,049 INFO    MainThread:17724 [wandb_config.py:__setitem__():141] config set dropout = 0.3 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002267D4E8C70>>
2021-09-22 16:19:22,049 INFO    MainThread:17724 [wandb_run.py:_config_callback():843] config_cb dropout 0.3 None
2021-09-22 16:19:22,050 INFO    MainThread:17724 [wandb_config.py:__setitem__():141] config set batch_size = 14 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002267D4E8C70>>
2021-09-22 16:19:22,050 INFO    MainThread:17724 [wandb_run.py:_config_callback():843] config_cb batch_size 14 None
2021-09-22 16:19:22,051 INFO    MainThread:17724 [wandb_config.py:__setitem__():141] config set pool_size = [6, 6] - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002267D4E8C70>>
2021-09-22 16:19:22,051 INFO    MainThread:17724 [wandb_run.py:_config_callback():843] config_cb pool_size [6, 6] None
2021-09-22 16:19:22,052 INFO    MainThread:17724 [wandb_config.py:__setitem__():141] config set shear_range = 0.4 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002267D4E8C70>>
2021-09-22 16:19:22,052 INFO    MainThread:17724 [wandb_run.py:_config_callback():843] config_cb shear_range 0.4 None
2021-09-22 16:19:22,052 INFO    MainThread:17724 [wandb_config.py:__setitem__():141] config set zoom_range = 0.4 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002267D4E8C70>>
2021-09-22 16:19:22,053 INFO    MainThread:17724 [wandb_run.py:_config_callback():843] config_cb zoom_range 0.4 None
2021-09-22 16:47:14,184 INFO    MainThread:17724 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-22 16:47:14,186 INFO    MainThread:17724 [wandb_run.py:_restore():1543] restore
